lib/resources/datav2/entityv2.js

```diff
@@ -32,30 +32,55 @@ const bar_mapping_v2 = {
   t: "Timestamp",
 };
 
+const snapshot_mapping_v2 = {
+  latestTrade: "LatestTrade",
+  latestQuote: "LatestQuote",
+  minuteBar: "MinuteBar",
+  dailyBar: "DailyBar",
+  prevDailyBar: "PrevDailyBar",
+};
+
 function AlpacaQuoteV2(data) {
-  return convert(data, quote_mapping_v2);
+  return convert(data, quote_mapping_v2, false);
 }
 
 function AlpacaTradeV2(data) {
-  return convert(data, trade_mapping_v2);
+  return convert(data, trade_mapping_v2, false);
 }
 
 function AlpacaBarV2(data) {
-  return convert(data, bar_mapping_v2);
+  return convert(data, bar_mapping_v2, false);
+}
+
+function AlpacaSnaphotV2(data) {
+  return convert(data, snapshot_mapping_v2, true);
 }
 
-function convert(data, mapping) {
+function convert(data, mapping, isSnapshot) {
   const obj = Object.fromEntries(
     Object.entries(data).map(([key, val]) => [
       mapping.hasOwnProperty(key) ? mapping[key] : key,
-      val,
+      isSnapshot ? convertSnapshotsVal(key.toLowerCase(), val) : val,
     ])
   );
   return obj;
 }
 
+function convertSnapshotsVal(key, data) {
```

Comment:
Please add tests to this function + I would refact to have explicit matching on keys - and maybe use switch() rather than many elseifs.